### PR TITLE
Network march release - ready for BUILD!!

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/applicationGateway.json [hunk 1]
```diff
@@ -0,0 +1,2658 @@
...
+    "ApplicationGatewayRewriteRuleCondition": {
+      "properties": {
+        "variable": {
+          "type": "string",
+          "description": "The condition parameter of the RewriteRuleCondition."
+        },
+        "pattern": {
+          "type": "string",
+          "description": "The pattern, either fixed string or regular expression, that evaluates the truthfulness of the condition"
+
```
Comment (nschonni) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/applicationGateway.json]:
```suggestion
          "description": "Properties of the application gateway SSL predefined policy."
```

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+            "BeginsWith",
+            "EndsWith",
+            "Regex"
+          ],
+          "x-ms-enum": {
+            "name": "WebApplicationFirewallOperator",
+            "modelAsString": true
+          }
+        },
+        "negationConditon": {
```
Comment (nschonni) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json]:
```suggestion
        "negationCondition": {
```

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/networkInterface.json [hunk 3]
```diff
@@ -0,0 +1,1364 @@
...
+    },
+    "IPConfigurationPropertiesFormat": {
+      "properties": {
+        "privateIPAddress": {
+          "type": "string",
+          "description": "The private IP address of the IP configuration."
+        },
+        "privateIPAllocationMethod": {
+          "$ref": "./network.json#/definitions/IPAllocationMethod",
+          "description": "The private IP adress allocation method."
```
Comment (nschonni) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/networkInterface.json]:
```suggestion
          "description": "The private IP address allocation method."
```

**Comment by MikhailTryakhov** [comment]
would open from another branch to apply fixes

**Pull request closed without merging**